rust/routee-compass/src/app/compass/compass_json_extensions.rs

```diff
@@ -7,9 +7,9 @@ pub trait CompassJsonExtensions {
 
 impl CompassJsonExtensions for serde_json::Value {
     /// attempts to read queries from the user in the three following ways:
-    ///   1. top-level of JSON is an array -> return it directly
```

Comment:
victims of find-and-replace on the word "return", can you _return_ these to how they were?

Reply:
whoops! yes good catch